app/assets/javascripts/discourse/tests/acceptance/user-card-test.js

```diff
@@ -26,3 +26,49 @@ acceptance("User Card - Show Local Time", function (needs) {
     );
   });
 });
+
```

Comment:
At first I wasn't sure how write a test to capture this issue - ensuring that clicking `@sAm` would display the correct capitalization on the user card - but I think these tests (hopefully) capture the essence of this issue (displaying the `username` on the user card). :sweat_smile: I factored in the setting `prioritize_username_in_ux` since it determines where the username will appear.

Side note: i was almost certain that this was a bug:
https://github.com/discourse/discourse/blob/794d2dabf6b2a0586e4918a226ab1a220064890f/app/assets/javascripts/discourse/app/templates/components/user-card-contents.hbs#L52-L53

I didn't understand why "if nameFirst then display username" was correct, but I realized it is to reverse what was displayed earlier (if the `name` is displayed first, then display `username` and vice versa). Hopefully these tests will make that a little clearer to a future reader. :crossed_fingers: :smile: